scvid/module/onepass_mean_var_std.py

```diff
@@ -36,9 +36,9 @@ def forward(self, x_ng: torch.Tensor) -> None:
         _, num_replicas = get_rank_and_num_replicas()
         if num_replicas > 1:
             x_ng = torch.cat(GatherLayer.apply(x_ng), dim=0)
-        self.x_sums += x_ng.sum(dim=0)
-        self.x_squared_sums += (x_ng**2).sum(dim=0)
-        self.x_size += x_ng.shape[0]
+        self.x_sums = self.x_sums + x_ng.sum(dim=0)
```

Comment:
Fixes runtime error (due to Python 3.10?): `RuntimeError: output with shape [] doesn't match the broadcast shape [5]`.

Reply:
Strange :)